dbt_schema_builder/schema.py

```diff
@@ -68,6 +94,18 @@ def filter_relations(self):
                 )
         return filtered_relations
 
+    def has_soft_delete_predicate(self):
```

Comment:
is this used anywhere? I tried searching for it and didn't find anything

Reply:
Ah nope, it got removed at the last minute. I'll remove.